armi/reactor/tests/test_components.py

```diff
@@ -1262,6 +1400,13 @@ def test_thermallyExpands(self):
 
 
 class TestRadialSegment(TestShapedComponent):
+    """Test radial segment.
+
+    .. test:: Compute properties of radial segment
+        :id: T_ARMI_COMP_FLUID0
+        :tests: R_ARMI_COMP_FLUID
+    """
+
```

Comment:
Again, delete this.  Or move it down to one-or-more of the tests in this class.

We can't have class-level test crumbs. Not your fault, you didn't know.